src/senaite/sync/importstep.py

```diff
@@ -168,6 +168,8 @@ def _import_users(self):
                 logger.debug("Skipping existing user {}".format(username))
                 continue
             email = user.get("email", "")
+            if not email:
+                email = "example@example.com"
```

Comment:
Please, add at least a comment why you need to setup a "fake" email to the user. As per the accepted answer here https://stackoverflow.com/questions/1368163/is-there-a-standard-domain-for-testing-throwaway-email , seems that using `example.com` as domain is fine for testing purposes. I'd just only do the following change:
```python
email = "{}@example.com".format(username)
```